CMakeLists.txt

```diff
@@ -140,13 +145,12 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cutelystqt5-config.cmake
     DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/cutelystqt5/
 )
 install(EXPORT CutelystQt5Targets
-    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/cutelystqt5/"
+    DESTINATION cmake
```

Comment:
Well, after successfully compiling and installing Grantlee on Windows, when running from the start the CMake build configuring:

PLUGIN: View::Grantlee, disabled.

I think this is because they didn't export(PACKAGE Grantlee)